src/Pages/addPhoto/AddPhoto.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import './AddPhoto';
+import Header from '../../Components/Header/header';
+import './AddPhoto.css';
+import { storage , store , timestamp  } from '../../Components/firebase';
+import LoaderModel from '../../Components/Loader/LoaderModal';
+import Button from '@material-ui/core/Button';
+import AddIcon from '@material-ui/icons/AddAPhoto';
+import DeleteIcon from '@material-ui/icons/DeleteForeverRounded';
+const types = ['image/png' , 'image/jpeg'];
+
+
+class AddPhoto extends React.Component {
+    constructor(props) {
+        super(props);
+         this.state = { 
+             pictures: [] ,
+             image : '',
+             url : '',
+             imageId:null,
+             isUpload:null,
+             isDelete:null
+        };
+    }
+
+
+      uploadImg = (e) => {
+        
+        const selected = e.target.files[0];
+        if(e.target.files[0] && types.includes(selected.type)){
```

Comment:
need to add error handling while accessing first element of the files array. You can use optional chaining for that like below:
`e.target.files?.[0]`

This will check if files exist or not and then It will check 0th element.